languages/csharp/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,57 @@
+C# is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by letting the C# compiler infer its type based on the assigned value (known as _type inference_). Therefore, the following two variable definitions are equivalent:
+
+```csharp
+int fingers = 10;
+var fingers = 10;
+```
+
+Updating a variable's value is done through the `=` operator:
+
+```csharp
+var count = 5;
+count = 7;
+```
+
+C# is an object-oriented language and requires all functions to be defined in a _class_. The `class` keyword is used to define a class. It is common to specify an _access modifier_, which influences what other classes can use the class. The most common access modifier is `public`, which means that other classes can use this class.
```

Comment:
I suggest that we hyperlink "an object-oriented language". I think previously we suggested linking to our reference docs for now, then doing a review of that at the end and writing an article for everything that's written, but you might like to link out to something now if you have a particuarl resource in mind instead.

Reply:
This is a really nice paragraph.

Reply:
Link added.